### DISCUSSION TITLE
记录一个颇为奇怪的问题

### DISCUSSION THREAD

**Discussion by yours57** [MetaCubeX/ClashMetaForAndroid#221] [Q&A]
使用cmfa一段时间后，对nekogram的代理就会失效。
与此同时，其他app可以正常访问。
重启后可以解决该问题。
但是使用一段时间后又会重现。
期待有相似经历的朋友分享解决方法。

**Comment by PingZi-Wing** [comment]
我这里原生安卓14，电报官方客户端直接不通，play商店、浏览器是正常的。其中报过一次：Start TUN listening error: create NetworkUpdateMonitor: netlink socket in Android is banned by Google, use the root or system (ADB) user to run sing-box, or switch to the sing-box Adnroid graphical interface client。看起来meta在借鉴sing box，可能安卓版本的meta内核太旧了吧。折腾半天，还是回归简单稳定的v2rayng了。
